src/components/AdminDashboard.jsx

```diff
@@ -88,13 +88,22 @@ class AdminDashboard extends React.Component {
         path: "incoming",
         role: "SUPERVOLUNTEER"
       },
+      {
+        name: "Tags",
+        path: "tags",
+        role: "SUPERVOLUNTEER"
+      },
       {
         name: "Settings",
         path: "settings",
         role: "SUPERVOLUNTEER"
       }
     ];
 
+    if (window.SHOW_TAGS === false) {
+      sections = sections.filter(section => section.name !== "Tags");
+    }
```

Comment:
See above, since this is probably just a temporary variable, I thought just being able to remove the false conditional when we no longer need it would be easier. But I'm open to changing it if it seems messy. @ibrand, we'll probably just remove this env. variable when we get the texter tag ui set to go,  right?

Reply:
If we are removing that if block anyway then we can leave it as is. Thanks!